src/components/Swap/Input.tsx

```diff
@@ -122,7 +151,7 @@ export default function Input() {
   }, [input, max, updateInputAmount])
 
   return (
-    <InputColumn gap={0.5} approved={mockApproved}>
+    <InputColumn gap={0.5} approved={mockApproved} disableHover={isDisabled}>
```

Comment:
This will need to be replicated in Output.tsx.
Also, the logic in both cases should be `disableHover={isDisabled || !swapXputCurrency}` - in other words, if no currency is selected, they should not be selectable.